AppAuth.podspec

```diff
@@ -62,5 +61,12 @@ It follows the OAuth 2.0 for Native Apps best current practice
     externalUserAgent.osx.deployment_target = '10.9'
   end
   
-  s.default_subspec = 'Core', 'ExternalUserAgent'
+  s.subspec 'EnterpriseUserAgent' do |enterpriseUserAgent|
+    enterpriseUserAgent.dependency 'AppAuth/Core'
+    
+    enterpriseUserAgent.source_files = "Source/AppAuthEnterpriseUserAgent.h", "Source/AppAuthEnterpriseUserAgent/iOS/OIDExternalUserAgentIOSCustomBrowser.{h,m}"
```

Comment:
Should it be this?
` enterpriseUserAgent.ios.source_files = "Source/AppAuthEnterpriseUserAgent/iOS/**/*.{h,m}"`

Reply:
Good point, that will allow for future extensibility. I think we should still include the header file explicitly